setup.py

```diff
@@ -54,13 +54,21 @@
         "WTForms==2.2.1",
         "esprit @ git+https://github.com/CottageLabs/esprit.git@edda12177effa0945d99302f0d453b22503e335b#egg=esprit",
         "dictdiffer @ git+https://github.com/CottageLabs/dictdiffer.git@cc86c1ca1a452169b1b2e4a0cb5fc9e6125bc572#egg=dictdiffer",
-        "flask-swagger @ git+https://github.com/DOAJ/flask-swagger.git@f1dbf918d9903a588eed3cce2a87eeccc9f8cc0e#egg=flask-swagger"
+        "flask-swagger @ git+https://github.com/DOAJ/flask-swagger.git@f1dbf918d9903a588eed3cce2a87eeccc9f8cc0e#egg=flask-swagger",
+
+
+        # pri list
+        'gspread~=5.10.0',
+        'oauth2client~=4.1.3',
+        'pandas~=2.0.1',# pandas lets us generate URLs for linkcheck
+        'gspread-dataframe~=3.3.1',
+        'gspread-formatting~=1.1.2',
```

Comment:
I think for these extra features, we shouldn't have them in the main requirements, as they create overhead for the production app install.  Suggest we have an `extras_require` for this feature, so that they only need to be installed for someone who is going to run this script